dbt/adapters/databricks/connections.py

```diff
@@ -2,13 +2,17 @@
 from dataclasses import dataclass, field
 import re
 import time
-from typing import Any, ClassVar, Dict, Iterator, List, Optional, Sequence, Tuple
+from typing import Any, Callable, ClassVar, Dict, Iterator, List, Optional, Sequence, Tuple
+
+import agate
```

Comment:
nit: up to you on style but `from agate import Table` to indicate we're only using a single type from the module.